DOI.js

```diff
@@ -101,14 +104,10 @@ function detectWeb(doc, url) {
 	return false;
 }
 
-function completeDOIs(doc) {
+function completeDOIs(_doc) {
 	// all DOIs retrieved now
 	// check to see if there is more than one DOI
-	var numDOIs = 0;
-	for (var DOI in selectArray) {
-		numDOIs++;
-		if (numDOIs == 2) break;
-	}
+	var numDOIs = Object.keys(selectArray).length;
```

Comment:
did you understand what was going on here? Why do we break at 2 and not at 1 above?

Reply:
I guess not then? I thought it was for performance reasons. But come to think of it, then 1 would have sufficed.

Reply:
The reason I went there at all is that the linter complained.

Reply:
yup, I know, and it should be possible to do this without the while loop either way (if we check for length>1 we can do that) but I want to know what's going on here before changing it.
@zuphilip , do you happen to remember?